lesson06/assignment/accordion.html

```diff
@@ -0,0 +1,138 @@
+                       <h2>
+                            <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#collapseThree">
+                                Where should I plant my tulip bulbs?
+                           </button>
+                       </h2>
+                    </div>
+
+                    <!--Third collapsible content-->
+                    <div id="collapseThree" class="collapse" data-parent="#accordion">
+                        <div class="card-body">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                        </div>
+                    </div>
+                </div>
+
+                <!--Fourth FAQ-->
+                <div class="card">
+                    <div class="card-header" id="headingFour">
+                        <h2>
+                            <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#collapseFour">
+                                How long do fresh cut tulips last?
+                            </button>
+                        </h2>
+                    </div>
+
+                    <!--Fourth collapsible content-->
+                    <div id="collapseFour" class="collapse" data-parent="#accordion">
+                        <div class="card-body">
+                           Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                        </div>
+                    </div>
+                </div> 
+            
+                <!--Fifth FAQ-->
+                <div class="card">
+                    <div class="card-header" id="headingFive">
+                        <h2>
+                            <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#collapseFive">
+                                What sets Tulip Fever tulips above the rest?
+                            </button>
+                        </h2>
+                    </div>
+
+                    <!--Fifth collapsible content-->
+                    <div id="collapseFive" class="collapse" data-parent="#accordion">
+                       <div class="card-body">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+        
+        <div style="clear: both;"></div>
```

Comment:
Embedding styles in an HTML element is not acceptable practice. All necessary styles should be applied using CSS. Also, this doesn't seem like a necessary item. Are you using floats on this page? Floats are rarely used anymore; flexbox solves most of the problems we used to use floats for, without requiring the "clear" cleanup.